### ISSUE TITLE
Using network-media-library/site_id filter to return main site (id 1) interferes with user capabilities

### ISSUE THREAD

**Issue by adriantoll** [humanmade/network-media-library#40]
I'd like all sites to use the media library of the main site in the network. To do this I used the `network-media-library/site_id` filter like this:

```
add_filter( 'network-media-library/site_id', function( $site_id ) {
    return 1;
} );
```

This seemed to work, in that media from the main site turned up on other sites, but it appears that it causes users on the main sites with a level lower than Super Admin to lose most of their capabilities (e.g. Editors can't edit Pages). I've tried removing from that site and re-adding them, but no luck.

Is there a specific way that I should be selecting the main site as the main media library, or is that not a use case this plugin covers for some reason?

**Comment by johnbillion** [comment]
This is an intentional design decision that I've since changed my mind about but haven't fixed yet. The original intention was that a user should have no need to perform any action on the central media site that doesn't relate to media management, but of course that's not necessarily true.

If you want to remove this "feature", you can comment out the following line: https://github.com/humanmade/network-media-library/blob/99615f80c8c11ff6f80eb86e70b1f8f5c252c789/network-media-library.php#L114

I'll remove this unnecessary restriction in a future update.

**Comment by adriantoll** [comment]
Thanks for the quick response @johnbillion

**Comment by johnbillion** [comment]
Fixed in version 1.4.0.
